### PR TITLE
Lobby page - Edit issue modal window

### PR DESCRIPTION
**Pull request by HelenBassa**
***Lobby page Edit Issue modal window***
edit-issue-modal
In scope:
- [x] issue card: 
on edit issue icon button click should appear a "Edit issue" modal window like "Create Issue" modal, with the same React controlled fields (Title, Link, Priority) and cancel / save buttons

### CODE REVIEW

File: client/src/components/modals/edit-issue-modal/edit-issue-modal.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import { ChangeEvent, useState, useEffect } from 'react';
+import { InputLabel, MenuItem, Select, TextField } from '@material-ui/core';
+import { IEditIssueModalErrors, IssuePriority } from '@models/types';
+import { ModalWrapper } from '../modal-wrapper/modal-wrapper.tsx';
+import { issueData } from './edit-issueData';
+import './edit-issue-modal.sass';
+
+const EditIssueModal = ({ isOpen, onClose }: any) => {
+  const [inputSettings, setInputSettings] = useState({
```
Comment (aksiantsiuk) [inline client/src/components/modals/edit-issue-modal/edit-issue-modal.tsx]: `any` type? Dont forget to define appropriate interface

File: client/src/components/modals/edit-issue-modal/edit-issue-modal.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+      <div className="text-input-issue-modal">
+        <div className="input-modal">
+          <TextField
+            label="Title"
+            name="titleIssue"
+            autoComplete="off"
+            fullWidth
+            value={inputSettings.titleIssue}
+            onChange={handleInput}
+            error={errors.titleIssueError === true}
```
Comment (aksiantsiuk) [inline client/src/components/modals/edit-issue-modal/edit-issue-modal.tsx]: `error={errors.titleIssueError}`

File: client/src/components/modals/edit-issue-modal/edit-issue-modal.tsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        </div>
+        <div className="input-modal">
+          <TextField
+            label="Link"
+            name="linkIssue"
+            autoComplete="off"
+            fullWidth
+            value={inputSettings.linkIssue}
+            onChange={handleInput}
+            error={errors.linkIssueError === true}
```
Comment (aksiantsiuk) [inline client/src/components/modals/edit-issue-modal/edit-issue-modal.tsx]: The same

**Review by RealOneReal — APPROVED** [review]

File: client/src/components/modals/edit-issue-modal/edit-issue-modal.tsx (hunk 1 above)
Comment (HelenBassa) [reply client/src/components/modals/edit-issue-modal/edit-issue-modal.tsx]: Thanks, fixed - define ***type IModalWindow = Pick<IModalWrapper, 'isOpen' | 'onClose'>***

File: client/src/components/modals/edit-issue-modal/edit-issue-modal.tsx (hunk 2 above)
Comment (HelenBassa) [reply client/src/components/modals/edit-issue-modal/edit-issue-modal.tsx]: Thanks, fixed

File: client/src/components/modals/edit-issue-modal/edit-issue-modal.tsx (hunk 3 above)
Comment (HelenBassa) [reply client/src/components/modals/edit-issue-modal/edit-issue-modal.tsx]: Thanks, fixed

**Review by aksiantsiuk — APPROVED** [review]

**Review by utrennik — APPROVED** [review]

**Pull request merged**